# AuvSetpoint.msg
float64 x      # [m]
float64 y      # [m]
float64 z      # [m]
float64 yaw    # [rad]
int32  mode    # 0: 절대(odom 기준), 1: 상대(현재 위치 기준)
